client/python/tensorflow-serving-api/samples/README.md

```diff
@@ -28,7 +28,8 @@ pip3 install -r requirements.txt
 Access to Google Cloud Storage might require proper configuration of https_proxy in the docker engine or in the docker container.
 In the examples listed below, OVMS can be started using a command:
 ```bash
-docker run -d --rm -e "http_proxy=$http_proxy" -e "https_proxy=$https_proxy" -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path gs://ovms-public-eu/resnet50 --port 9000 --rest_port 8000
+wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.{xml,bin} -P models/resnet50/1
+docker run -d -u $(id -u) -v $(pwd)/models:/models -p 8000:8000 -p 9000:9000 openvino/model_server:latest --model_name resnet --model_path /models/resnet50 --port 9000 --rest_port 8000
```

Comment:
Why `-u $(id -u)`? Why removed `--rm`?

Reply:
-u to ensure access to models regardless of the user mask, removed -d just to make the command shorter

Reply:
By removing `--rm` the user will have hard time executing this command multiple times when learning to use OVMS

Reply:
what kind of problems do you expect? if left containers on the disk is a concern user could add more params as needed

Reply:
Re-running the command will not be possible due to port already taken

Reply:
rm option just removes the container after it is stopped. It doesn't avoid port blocking. It is just a way to cleanup the containers after they are used.

Reply:
Tested and re-running this command (or any other with `-d`) causes:
```
Bind for 0.0.0.0:9700 failed: port is already allocated.
```
Reverting back to `-it --rm` to make user be able to jump to any part of the documentation without a need to manually search for started container name and stopping it manually